bindings/pyroot_experimental/PyROOT/src/PyzPythonHelpers.cxx

```diff
@@ -41,7 +41,7 @@ PyObject *PyROOT::GetSizeOfType(PyObject * /*self*/, PyObject *args)
    // Call interpreter to get size of data-type using `sizeof`
    long size;
    std::stringstream code;
-   code << "*((long*)" << &size << ") = (long)sizeof(" << dtype << ")";
+   code << "*((long*)" << std::hex << std::showbase << (size_t)&size << ") = (long)sizeof(" << dtype << ")";
```

Comment:
This does not change anything on Linux?

Reply:
I don't think so...

Reply:
(this is the goal of this PR)

Reply:
@etejedor I just tested, it doesn't change anything on Linux